tools/d2g/d2g.go

```diff
@@ -295,6 +296,14 @@ func setMaxRunTime(dwPayload *dockerworker.DockerWorkerPayload, gwPayload *gener
 
 func setOnExitStatus(dwPayload *dockerworker.DockerWorkerPayload, gwPayload *genericworker.GenericWorkerPayload) {
 	gwPayload.OnExitStatus.Retry = dwPayload.OnExitStatus.Retry
+
+	// An error sometimes occurs while pulling the docker image:
+	// Error: reading blob sha256:<SHA>: Get "<URL>": remote error: tls: handshake failure
+	// And this exits 125, so we'd like to retry.
+	if !slices.Contains(gwPayload.OnExitStatus.Retry, 125) {
```

Comment:
The implementation in slices is pretty trivial: https://cs.opensource.google/go/x/exp/+/fffb1438:slices/slices.go;l=106-113

As per the go proverb [_a little copying is better than a little dependency_](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s) I think I would prefer it if we just looped through the slice here and didn't import the full slices package.